apps/rollups/src/containers/rollups/DApp.tsx

```diff
@@ -306,14 +311,90 @@ export const EpochItem: FC<ItemProps<Epoch>> = ({ item }) => {
     );
 };
 
-export const DApp = () => {
-    const [, setSearch] = useState<string>('');
+interface NodeProps<T> {
+    node: T;
+}
+
+type Node = {
+    index: number;
+    notices: {
+        totalCount: number;
+        pageInfo: any;
+        edges: Edge<Notice>[];
+    };
+    reports: {
+        totalCount: number;
+        pageInfo: any;
+        edges: Edge<Report>[];
+    };
+    vouchers: {
+        totalCount: number;
+        pageInfo: any;
+        edges: Edge<Voucher>[];
+    };
+};
+
+const InputEdgeItem: FC<NodeProps<Node>> = ({ node }) => {
+    const bg = useColorModeValue('white', 'gray.800');
+    return (
+        <VStack bg={bg} alignItems="flex-start">
+            <Box
+                width="100%"
+                textAlign="center"
+                bg="blue.100"
+                py={2}
+                color="black"
+            >
+                <Heading fontSize="2xl">Input {node.index}</Heading>
+            </Box>
+            <Box px={3} py={2} width="full">
+                <SimpleGrid columns={{ base: 1 }}>
+                    <InputContent
+                        count={node.notices.totalCount}
+                        items={node.notices.edges}
+                        label="Notice"
+                    />
+
+                    <InputContent
+                        count={node.reports.totalCount}
+                        items={node.reports.edges}
+                        label="Report"
+                    />
+
+                    <InputContent
+                        count={node.vouchers.totalCount}
+                        items={node.vouchers.edges}
+                        label="Voucher"
+                    />
+                </SimpleGrid>
+            </Box>
+        </VStack>
+    );
+};
+
+export interface DAppProps {
+    address: string;
+    chainId: number;
+}
+
+export const DApp: FC<DAppProps> = (props) => {
+    const { address, chainId } = props;
+    const [search, setSearch] = useState<string>('');
     const bg = useColorModeValue('gray.80', 'header');
     const [result] = useDappQuery({
         variables: {},
     });
-    const { data, fetching, error } = result;
+    const [inputEdge] = useInputEdgeQuery({
+        variables: {},
+    });
 
+    const [dappVersionResult] = useDappVersionQuery({
```

Comment:
@dandheedge, 
I believe that is not going to the intended graphQL server (i.e. Subgraph). The reason being because of the provider around the DApp.tsx [here](https://github.com/cartesi/explorer/blob/86e759d6bb0bd2c73a4cd6ed1ff8cbadc72abfd4/apps/rollups/src/pages/dapp/%5Baddress%5D.tsx#L62) so by react tree distance and which provider is closer the query hook will use that provider and that query does not exist.

Reply:
In addition, that will not work as expected when the `DAPP_LOCAL_DEV` is on because, in that mode, the Subgraph dependency is ruled out. In a high-level way, I would evaluate using a custom hook that contains that generated hook + something related to the useApplications hook that currently has logic to help the main page to load information when that mode is on.

Reply:
with the latest changes we will do i.e. drop support to 0.8 that comment can be ignored.